technologies_deployments.md

```diff
@@ -0,0 +1,86 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+{% include JB/setup %}
+<div class="tech_deploy">
+  <div class="tech_deploy container">
```

Comment:
@soralee What i mentioned is, using different name in different `DIV`s. for example

```HTML
<div class=tech-deploy-outbox>
  <div class=tech-deploy-innerbox>
```

not using the same class name `tech_deploy`.